python/paddle/fluid/dygraph/parallel.py

```diff
@@ -43,13 +48,14 @@ def prepare_context(strategy=None):
     place = framework._current_expected_place()
```

Comment:
use new device api?

Reply:
same above, using new api is not efficient